src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -171,11 +171,12 @@ void regmodule_properties(py::module m) {
 
     // Special case: ov::device::PCIInfo
     py::class_<ov::device::PCIInfo, std::shared_ptr<ov::device::PCIInfo>> cls_pciinfo(m_device, "PCIInfo");
-    cls_pciinfo.def(py::init<const uint32_t&, const uint32_t&, const uint32_t&, const uint32_t&>());
+    cls_pciinfo.def(py::init<const uint32_t&, const uint32_t&, const uint32_t&, const uint32_t&, const uint32_t&>());
```

Comment:
Won't adding a new `init` argument with no default value break existing scripts? PyAPI needs to follow strict backwards compatibility rules. Is it possible to have a placeholder value here?

Reply:
True, I just wanted to make sure we're aware about this. :)